READMEs/install.md

```diff
@@ -44,12 +44,20 @@ pip3 install --pre ocean-lib
 #### Create your network configuration file
 
 ocean.py uses brownie to connect to deployed smart contracts. To configure the RPC URLs, gas prices and other settings,
-create and fill in a network-config.yml in your `~/.brownie` folder, or copy our (very basic) sample from ocean.py:
-
-```console
-mkdir ~/.brownie
-cp network-config.sample ~/.brownie/network-config.yaml
+create and fill in a network-config.yaml in your `~/.brownie` folder.
```

Comment:
This is automatically created when installing brownie. I think we should remove the 'create' statement